test/js-native-api/6_object_wrap/myobject.h

```diff
@@ -1,5 +1,5 @@
-#ifndef TEST_JS_NATIVE_API_6_OBJECT_WRAP_MYOBJECT_H_
```

Comment:
I wonder what is the proper conventions for the header ifdefs? E.g. I see that all header files in the `src` folder are prefixed with `SRC_`. Why not to do the same in the `test` folder?